source/uwp/Renderer/lib/AdaptiveFact.cpp

```diff
@@ -63,6 +77,19 @@ AdaptiveNamespaceStart
 
     HRESULT AdaptiveFact::GetSharedModel(std::shared_ptr<AdaptiveSharedNamespace::Fact>& sharedModel)
     {
+        std::shared_ptr<AdaptiveSharedNamespace::Fact> fact = std::make_shared<AdaptiveSharedNamespace::Fact>();
+
+        std::string value;
+        RETURN_IF_FAILED(HStringToUTF8(m_value.Get(), value));
+        fact->SetValue(value);
+
+        std::string language;
+        if (!(WindowsIsStringEmpty(m_language.Get())))
+        {
+            RETURN_IF_FAILED(HStringToUTF8(m_language.Get(), language));
+            fact->SetLanguage(language);
+        }
+
         sharedModel = m_sharedFact;
```

Comment:
The behavior won't be correct here. You're setting the value and language on the member variable fact, but you're not actually returning it (you're returning m_sharedFact instead). Also it looks like put_Value and get_Value are using m_sharedFact rather than the new member variable you added. As mentioned above, either add a m_Title and get rid of m_sharedFact altogether, or else get rid of m_value and m_language and get everything from m_sharedFact directly.